circuit_knitting/cutting/cutqc/__init__.py

```diff
@@ -33,3 +35,12 @@
     "create_dd_bin",
     "reconstruct_dd_full_distribution",
 ]
+
+warn(
+    f"The package {__name__} is deprecated and will be removed no sooner than Circuit Knitting Toolbox 0.8.0. "
+    "Use wire cutting modules in the circuit_knitting.cutting package for wire cutting."
```

Comment:
@garrison is the phrasing used here okay by you?

Reply:
There is a wire cutting implementation in `circuit_knitting.cutting`, but I am not sure what "wire cutting modules" refers to.

There is also the automatic cut finding, which is currently not mentioned by this string.

But I think it makes sense to be even more vague: There is a new circuit cutting implementation in `circuit_knitting.cutting` that implements similar and improved functionality, which will be maintained going forward.

Reply:
Oh, I did mention automatic cut finding in the line below :). But your overall suggestion sounds good.

Reply:
> Oh, I did mention automatic cut finding in the line below :)

Ahh, I knew I had seen it in an "earlier revision" :slightly_smiling_face: